students/Wieslaw_Pucilowski/lesson09/mailroom5_OO.py

```diff
@@ -191,17 +157,94 @@ def write_letters(self):
             d.write_letter()
 
 
+# Donors DB initialization
+dd = DonorsCollection()
+
+
+class Main:
+    @staticmethod
+    def report():
+        dd.print_report()
+
+    @staticmethod
+    def letters():
+        dd.write_letters()
+
+    @staticmethod
+    def donor():
+        dd.add_donor()
+
+    @staticmethod
+    def show():
+        dd.display()
+
+    def quit(msg):
```

Comment:
this is not a valid method. if you want to have a regular function inside a class then it needs to be a staticmethod.